geniza/common/admin.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
+
+
+class LocalUserAdmin(UserAdmin):
+    '''Extends :class:`django.contribut.auth.admin.UserAdmin`
+        to provide additional detail for user administration.'''
+    list_display = UserAdmin.list_display + (
+        'is_superuser', 'is_active', 'last_login', 'group_names')
+
+    def group_names(self, obj):
+        '''Custom property to display group membership.'''
+        if obj.groups.exists():
+            return ', '.join(g.name for g in obj.groups.all())
+    group_names.short_description = 'groups'
```

Comment:
this is a great idea!

Reply:
I copied it from one of our other projects — we haven't used it consistently, but I remember it being helpful when we did!